### PR TITLE
[MIG][17.0] website_sale_hide_price: Migration to 17.0

### PR DESCRIPTION
**Pull request by sylvainvh**

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Hi, @sylvainvh  Thanks for your contribution. As a general comment, you need to follow the migration guidelines so the module commit history is kept. You can check them here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

**Comment by sylvainvh** [comment]
> Hi, @sylvainvh Thanks for your contribution. As a general comment, you need to follow the migration guidelines so the module commit history is kept. You can check them here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Hi,

Sorry I missed this step. I force-pushed the branch with the history

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
The module works well, but need to run `pre-commit` command.

File: website_sale_hide_price/__manifest__.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Website Sale Hide Price",
-    "version": "16.0.2.0.0",
+    "version": "17.0.1.0.0",
     "category": "Website",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, Odoo Community Association (OCA), Idealis Consulting",
```
Comment (chienandalu) [inline website_sale_hide_price/__manifest__.py]: You can set yourself as contributor in the readme, but migration works (unless there's a substantial refactor of code) doesn't entitle authorship of the module

```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tested and looking good. Just fix pre-commit issues and this tiny change:

File: website_sale_hide_price/__manifest__.py (hunk 1 above)
Comment (sylvainvh) [reply website_sale_hide_price/__manifest__.py]: Hello, I made this modification and happen the modification on my previous commit

File: website_sale_hide_price/readme/CONTRIBUTORS.rst
Comment (chienandalu) [inline website_sale_hide_price/readme/CONTRIBUTORS.rst:1]: You can add yourself here

**Review by chienandalu — APPROVED** [review]
Thanks!

**Comment by chienandalu** [comment]
please review @pilarvargas-tecnativa

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Comment by pilarvargas-tecnativa** [comment]
LGTM thanks!

**Pull request merged**